ntoskrnl/ke/i386/mp.c

```diff
@@ -33,10 +41,10 @@ KeStartAllProcessors()
         ProcessorCount++;
 
         /* Attempt to allocate memory for its new setup */
-        SIZE_T APInfo = sizeof(KPCR) + sizeof(KTSS) + sizeof(ETHREAD) + GdtDesc.Limit + 1 + IdtDesc.Limit + 1; 
-        PVOID PAPInfo = MmAllocateContiguousMemory(APInfo, HighestPhysicalAddress);
+        SIZE_T APInfoSize = sizeof(struct APInfo) + 2;
```

Comment:
where does this `+2` come from ?

Reply:
GDT limit + 1 
IDT limit +1,  
You want a slight offset to the size of the table.

Reply:
What is that needed for? That should be inside the structure

Reply:
Yeah, I left a comment about that. adding 2 bytes and then copying the entire GDT into the descriptor doesn't really work. 😆